### PR TITLE
Sonata media

### PR DESCRIPTION
**Pull request by WhiteEagle88**

### CODE REVIEW

**Comment by aivus** [comment]
:-1:

**Comment by aivus** [comment]
@WhiteEagle88  = :shipit:

File: htdocs/app/config/routing.yml [hunk 1]
```diff
@@ -32,4 +32,12 @@ admin:
...
\ No newline at end of file
+    prefix: /admin
+
+gallery:
+    resource: '@SonataMediaBundle/Resources/config/routing/gallery.xml'
+    prefix: /media/gallery
+
+media:
+    resource: '@SonataMediaBundle/Resources/config/routing/media.xml'
+    prefix: /media
```
Comment (malavsky) [inline htdocs/app/config/routing.yml]: please, add new line to the end of file

File: htdocs/src/Application/UserBundle/Admin/UserAdmin.php [hunk 2]
```diff
@@ -11,11 +11,16 @@
 use Application\UserBundle\Entity\EntityManager\UserManager;
 use Application\UserBundle\Entity\User;
 
+/**
```
Comment (malavsky) [inline htdocs/src/Application/UserBundle/Admin/UserAdmin.php]: @aivus @WhiteEagle88 We need to decide, should we add phpDoc to Class? I think, if we use something special in this class we should add a phpDoc description about this class, other way should stay empty

**Comment by WhiteEagle88** [comment]
@malavsky 
Yeah, we add phpDoc to Class if we need describe some features

File: htdocs/src/Application/UserBundle/Admin/UserAdmin.php (hunk 2 above)
Comment (aivus) [inline htdocs/src/Application/UserBundle/Admin/UserAdmin.php]: I think we should remove class phpdocs which contains classname only

**Comment by aivus** [comment]
@WhiteEagle88  thanks

**Pull request merged**